index.html

```diff
@@ -1,38 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Joel DeLeon</title>
     <link rel="stylesheet" href="css/index.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Cedarville+Cursive&display=swap" rel="stylesheet">
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cedarville+Cursive&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
 </head>
+
 <body>
+
     <h1>Joel DeLeon</h1>
-<section id="About">
-    <h2>About</h2>
-    <p>My Name is Joel DeLeon, and im learnig about programming with code the dream.</p>
-</section>
-
-<section id="Experience">
-<h2>Experience</h2>
-<ul>
-    <li>Basketball Coach</li>
-    <li>PCA</li>
-    <li>delivery driver</li>
-</ul>
-</section>
-
-<section id="Connect">
-    <h2>Connect</h2>
-
-<a href="https://www.youtube.com/@Theloredude">YouTube</a>
-<i class="fab fa-youtube"></i>
-</section>
+    <nav>
+        <ul>
+            <li><a href="#About">About</a></li>
+            <li><a href="#Experience">Experience</a></li>
+            <li><a href="#Connect"></a>Connect</li>
+            <li><a href="#Skills"></a>Skills</li>
+            <li><a href="#Projects"></a>Projects</li>
```

Comment:
Your texts for these links are outside the `a` tag, but should be inside.

```suggestion
            <li><a href="#Connect">Connect</a></li>
            <li><a href="#Skills">Skills</a></li>
            <li><a href="#Projects">Projects</a></li>
```